ffi/capi/Cargo.toml

```diff
@@ -26,33 +26,47 @@ include = [
 [package.metadata.docs.rs]
 all-features = true
 
+[package.metadata.cargo-all-features]
+# Omit most optional dependency features from permutation testing
+skip_optional_dependencies = true
+
 [lib]
 crate-type = ["staticlib", "rlib", "cdylib"]
 path = "src/lib.rs"
 
+[features]
+default = ["plurals", "decimal", "locale_canonicalizer", "provider_fs", "provider_static"]
+wearos = ["decimal", "smaller_static", "freertos"]
+
+plurals = ["icu_plurals"]
+decimal = ["icu_decimal", "fixed_decimal"]
+locale_canonicalizer = ["icu_locale_canonicalizer"]
```

Comment:
Request: can we avoid adding feature flags for components?  That leads to an unbounded list of features in the future, which comes with a nontrivial maintenance cost.